src/frontend/src/flows/addDevice/addDeviceSuccess.ts

```diff
@@ -11,20 +11,28 @@ export type AddDeviceSuccessTemplateProps = Parameters<
 >[0];
 
 const addDeviceSuccessTemplate = ({
+  userNumber,
   deviceAlias,
   onContinue,
+  stepper,
   i18n,
 }: {
+  userNumber: bigint;
   deviceAlias: DeviceAlias;
   onContinue: () => void;
+  stepper?: TemplateResult;
   i18n: I18n;
 }) => {
-  const { title, continue_to_home, explore } = i18n.i18n(copyJson);
+  const copy = i18n.i18n(copyJson);
 
   const slot = html`<article>
+    ${stepper}
     <hgroup>
-      <h1 class="t-title t-title--main">${title}</h1>
-      <p class="t-lead">${explore}</p>
+      <div class="c-card__label">
+        <h2>${copy.internet_identity} ${userNumber}</h2>
```

Comment:
Can't you add or interpolate input into the i18n?

Maybe in other languages it would be `${userNumber} Internet Identity`.

Only if you already have the functionality. Otherwise, we can add it in another PR.

Reply:
Makes sense.